resources/views/pages/user/orders/index.blade.php

```diff
@@ -35,86 +35,144 @@
             <x-searchbar :filters="$filters" :action="route('user.orders.index')" />
         </div>
 
+        @if(session('success'))
+            <div class="bg-green-100 border border-green-400 text-green-700 px-4 py-3 rounded relative" role="alert">
+                <span class="block sm:inline">{{ session('success') }}</span>
+            </div>
+        @endif
+
+        @if(session('error'))
+            <div class="bg-red-100 border border-red-400 text-red-700 px-4 py-3 rounded relative" role="alert">
+                <span class="block sm:inline">{{ session('error') }}</span>
+            </div>
+        @endif
+
         <div class="overflow-x-auto w-full">
             <table class="w-full bg-[#FFF7FD] shadow rounded-lg overflow-hidden">
                 <thead class="bg-[#FFEBFA] text-gray-700 text-sm font-semibold">
                     <tr>
+                        <th class="px-4 py-2 text-left">Order ID</th>
                         <th class="px-4 py-2 text-left">Status</th>
                         <th class="px-4 py-2 text-left">Total Price</th>
+                        <th class="px-4 py-2 text-left">Event Date</th>
                         <th class="px-4 py-2 text-left">Created At</th>
                         <th class="px-4 py-2 text-left">Actions</th>
                     </tr>
                 </thead>
                 <tbody class="text-sm text-gray-800 divide-y divide-gray-200">
                     @forelse ($orders as $order)
+                        @php
+                            $event = optional($order->orderItems->first()->ticket->event);
+                            $eventDate = $event->date ?? null;
+                            $canCancel = in_array($order->status, ['paid', 'pending']) && $eventDate && $eventDate->isFuture();
+                            $canRequestRefund = $order->status === 'cancelled' && !$order->refund && $eventDate && $eventDate->isPast();
```

Comment:
Jesteś pewna że powinno być tutaj cancelled a nie na przykład paid?
```suggestion
                            $canRequestRefund = $order->status === 'paid' && !$order->refund && $eventDate && $eventDate->isPast();
```

Reply:
nie jestem pewna